### DISCUSSION TITLE
Tian Xiao

### DISCUSSION THREAD

**Discussion by xtt9** [cmadland/edci338-202201#46] [General]
#### What is the URL for the blog you created? You are welcome to use a blog that you've created for another class, or your own personal site. Just make sure you use the category `edci338` on your posts.

[Your Blog Title](https://xtt3.opened.ca)

- [x]  Check this box if you are willing to have your blog URL published on the course website.
- [x]  Check this box if you agree to have your blog posts syndicated so that they are displayed on the course site.

####  If you have a social media account that you would like to associate with EDCI338, feel free to enter a link to your profile here. I won't follow you unless you invite me, but, with your permission, I might tag you to share your work.
* Twitter: @xiaotian9859

- [x]  Check here if you give me permission to share your work publicly and tag you.

#### Please indicate which social media tools you are currently using and/or would like to learn more about.
* Instagram 
* TikTok

#### Please check the following boxes if you agree with the statement.

- [x]  I agree to share my work for EDCI338 on the open web.
- [x]  I agree to let my instructor publicize my work through his Twitter account (@colinmadland).
- [x]  Check this box if you are taking any other courses right now, and if so, indicate how many below.
    - [ ]  1 other course
    - [ ]  2 other courses
    - [ ]  3 other courses
    - [x]  4 other courses
    - [ ]  5 or more other courses
- [ ]  Do you have any plans that will require you to be away from the course for more than a few days? It's not necessarily a problem if you do, but it will be helpful to plan ahead. Also, I recognize that sometimes, things come up that you can't avoid. Please just let me know if you are going to be away. We can always work something out.

#### Please tell me a little bit about why you are taking EDCI338.
- [ ] As an elective in the Faculty of Education (you are a pre-service teacher)
- [x] As an elective in another faculty at UVic
- [ ] As an elective for a different university
- [ ] As an active teacher seeking professional development

**Comment by xtt9** [comment]
Hi, here is my first wow and wonder post!
https://xtt3.opened.ca/wow-and-wonder-1-pln/

**Comment by xtt9** [comment]
Hi, here is my second wow and wonder post!
https://xtt3.opened.ca/wow-and-wonder-2-connectivism/

**Comment by xtt9** [comment]
Hi, here is my showcase 1!
https://xtt3.opened.ca/showcase-post-1/

**Comment by cmadland** [comment]
Thank you!

**Comment by cmadland** [comment]
I've moved this back to be an issue. #109
